src/xml/report.html.xsl

```diff
@@ -1,62 +1,122 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
 
-<xsl:stylesheet 
-  xmlns:xsl="http://www.w3.org/1999/XSL/Transform" 
+<xsl:stylesheet
+  xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
   version="1.0"
   >
-  <xsl:output method="html"/> 
- 

```

Comment:
Ditto re: white space .... I'm not going to keep pointing it out. I'm sure you get the point .... it applies throughout.